### PR TITLE
Blackbox testing

### PR DESCRIPTION
**Pull request by licitdev**
Initial blackbox tests.

### CODE REVIEW

File: tests-blackbox/common/variables.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import { PrimaryKeyType } from './types';
```
Comment (rijkvanzanten) [inline tests-blackbox/common/variables.ts:1]: Should we rely on `import { PrimaryKey } from '@directus/shared/types';` instead?

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
Lets remove the `.gitkeep` files in favor of a single `todo.md` file (or issue) that lists all the tests we know we want/need to add

**Review by rijkvanzanten — COMMENTED** [review]
Tests themselves LGTM! 🚀 The one crucial thing that's missing as of right now is in-depth documentation on how to write these tests. I've added https://github.com/directus/docs/issues/39 to track that 👍🏻

File: tests-blackbox/common/variables.ts (hunk 1 above)
Comment (licitdev) [reply tests-blackbox/common/variables.ts:1]: Nope. The `PrimaryKeyType` consists of the following string literals `'integer' | 'uuid' | 'string'`.

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**